.github/workflows/CI.yml

```diff
@@ -0,0 +1,57 @@
+name: CI
```

Comment:
What about naming this workflow as something meaningful? 
`CI` seems too generic. Maybe `On Pull Request`?

Reply:
All right!